### DISCUSSION TITLE
Can i edit database records with Visidata?

### DISCUSSION THREAD

**Discussion by 4l1fe** [saulpw/visidata#1495] [Q&A] [answered]
Instead of creating a new sheet i'd like to save records change right into a database, is it possible?

I find vd a great tool to rapidly explore DB and edit some data on the go.

**Comment by saulpw** [comment, marked as answer]
Yes, if it's a database, the changes are deferred; commit them with `z Ctrl+S`.

**Reply by 4l1fe** [reply]
Great
